src/Symfony/Bundle/TwigBundle/DependencyInjection/Configuration.php

```diff
@@ -135,7 +136,7 @@ private function addTwigOptions(ArrayNodeDefinition $rootNode)
                 ->end()
                 ->scalarNode('autoescape_service')->defaultNull()->end()
                 ->scalarNode('autoescape_service_method')->defaultNull()->end()
-                ->scalarNode('base_template_class')->example('Twig\Template')->cannotBeEmpty()->end()
+                ->scalarNode('base_template_class')->example(Template::class)->cannotBeEmpty()->end()
```

Comment:
I would revert this one as this is an example, not a real class.

Reply:
Actually, a class name is expected here and the value of `Template::class === 'Twig\Template'`. I would make the change.